crates/bevy_app/src/app.rs

```diff
@@ -91,6 +96,18 @@ impl Debug for App {
 
 impl Default for App {
     fn default() -> Self {
+        #[cfg(not(target_arch = "wasm32"))]
+        {
+            static CTRL_C_HOOK: std::sync::Once = std::sync::Once::new();
+
+            CTRL_C_HOOK.call_once(|| {
+                ctrlc::set_handler(move || {
```

Comment:
Not a big fan of putting a side effect in a `Default` impl, let alone one that will overwrite any previously registered SIGINT handler from outside bevy.

Could this be a plugin in the `DefaultPlugins` group instead? That way it can be easily disabled, and won't interfere with `App`s created in unit tests.

Reply:
moved it to a plugin and added it to the `DefaultPlugins`, do you think it should be part of the minimal plugins?

Reply:
IMO no, this feature isn't needed for minimal bevy usage